recipes/shyaml/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "shyaml" %}
+{% set version = "0.5.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 80650ebfe6fa2e16083451d515207472d60990c1c15fc0fd607c27077790ac23
+  patches:
+    - cli-args.patch
```

Comment:
Could we add a comment above this about what it is needed for? Also has this been raised with upstream?

Reply:
I think this is an issue only for entry points, which the original package doesn't use but we have to.